api/README.md

```
@@ -1,101 +1,340 @@
+make deps                   # Install dependencies
+make proto                  # Generate type-safe contracts
+make dev                    # Start with hot reload 🔥
+
+# API ready at http://localhost:8080
+# Interactive docs at http://localhost:8080/docs
 ```
+
+## 🏗️ Architecture Philosophy
+
+### Pure Onion Architecture
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)